src/Juvix/Prelude/Base.hs

```diff
@@ -365,6 +365,11 @@ zip4Exact [] [] [] [] = []
 zip4Exact (x1 : t1) (x2 : t2) (x3 : t3) (x4 : t4) = (x1, x2, x3, x4) : zip4Exact t1 t2 t3 t4
 zip4Exact _ _ _ _ = error "zip4Exact"
 
+allEqual :: (Eq a) => [a] -> Bool
```

Comment:
This function exists in the `extra` library (which Juvix depends on), so we could use/export this instead.

https://hackage.haskell.org/package/extra-1.7.14/docs/Data-List-Extra.html#v:allSame

Reply:
Done